index.bs

```diff
@@ -1153,7 +1154,9 @@ The dictionary SHALL have the following attributes:
 :: The number of bytes sent on the QUIC connection, including retransmissions.
    Does not include UDP or any other outer framing.
 : <dfn for="WebTransportStats" dict-member>packetsSent</dfn>
-:: The number of packets sent on the QUIC connection, including retransmissions.
+:: The number of packets sent on the QUIC connection, including loss.
```

Comment:
```suggestion
:: The number of packets sent on the QUIC connection, including those that are determined to have been lost.
```

A citation to [Section 6 of RFC 9002](https://quicwg.org/base-drafts/rfc9002.html#name-loss-detection) might help explain the difference between "lost" and "determined to have been lost".